package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cyclops",
-  "version": "1.8.4",
-  "description": "cyclops: The UI/UX Pattern Guide",
+  "version": "1.9.0-beta.1",
```

Comment:
This is a major version change (not a minor) since the component is removed without an upgrade path.